### PR TITLE
Documentation for new integration, Add WiLight integration #35867

### PR DESCRIPTION
**Pull request by leofig-rj**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36694
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1642
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wilight.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Alternately, WiLight devices that are not discoverable can be statically configured. If you have WiLight devices on subnets other than where Home Assistant is running, or devices in a remote location reachable over a VPN, you will need to configure them manually. Statically configured devices may be used with or without automatic discovery enabled. Example static configuration:
+
+```yaml
+# Example configuration.yaml entry with automatic discovery disabled, and 2 statically configured devices
+wilight:
+  discovery: false
+  static:
+    - 192.168.1.23
+    - 192.168.52.172
+```
```
Comment (frenck) [inline source/_integrations/wilight.markdown]: As per ADR-0010, the YAML configuration should not be added.
See parent PR for more information.

Comment (leofig-rj) [reply source/_integrations/wilight.markdown]: New parent PR#36694 without YAML configuration. Only SSDP.
New documentation.

Comment (leofig-rj) [reply source/_integrations/wilight.markdown]: Now with SSDP only configuration. No more YAML.

Comment (leofig-rj) [reply source/_integrations/wilight.markdown]: Wilight initial with SSDP #36694

Comment (leofig-rj) [reply source/_integrations/wilight.markdown]: Now with only SSDP configuration.
New parent: Wilight initial with SSDP #36694

**Review by frenck — APPROVED** [review]
Thanks, @leofig-rj 👍

**Pull request merged**